tests/platforms/kubernetes/kubeobj.go

```diff
@@ -67,8 +67,9 @@ func buildDeploymentObject(namespace string, appDesc AppDescription) *appsv1.Dep
 				Spec: apiv1.PodSpec{
 					Containers: []apiv1.Container{
 						{
-							Name:  appDesc.AppName,
-							Image: fmt.Sprintf("%s/%s", appDesc.RegistryName, appDesc.ImageName),
+							Name:            appDesc.AppName,
+							Image:           fmt.Sprintf("%s/%s", appDesc.RegistryName, appDesc.ImageName),
+							ImagePullPolicy: "Always",
```

Comment:
Since an image tag is always the same, without this change updates to an application are not pulled on subsequent test calls.

Reply:
yeah makes sense! Thanks!